.github/workflows/format.yml

```diff
@@ -0,0 +1,23 @@
+name: Format
+on: pull_request
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Format
+      run: yarn install; yarn format
+    - name: Push changes
+      uses: kristoferbaxter/github-push-action@master
```

Comment:
For instance, you likely don't want to use this portion here that authors commits as me for prettier integration.

I'll remove this action and prettier in general from this PR.

Reply:
Removed prettier.